xapian-core/cluster/euclidian_sim.cc

```diff
@@ -50,6 +50,8 @@ EuclidianDistance::similarity(TermListGroup tlg,
 	LOGCALL(API, double, "EuclidianDistance::similarity()", "");
 	vector<pair<string, double> > docvec1;
 	vector<pair<string, double> > docvec2;
+	docvec1.clear();
+	docvec2.clear();
```

Comment:
This isn't fixing the tests, it's fixing the implementation. Should this entire commit be a fixup for earlier work?